frost-core/src/error.rs

```diff
@@ -53,7 +53,11 @@ pub enum Error {
     IdentityCommitment,
     /// Signature share verification failed.
     #[error("Invalid signature share.")]
-    InvalidSignatureShare,
+    InvalidSignatureShare {
+        /// The identifier of the signer whose share validation failed,
+        /// encoded as a little-endian byte string in hex format.
+        signer: String,
```

Comment:
If this is little-endian bytes, should it be just Vec<u8> rather than String? If String makes things easier that's a fine answer

Reply:
Good point, no particular reason to use a string.

I'm kinda having second thoughts about the endianess issue. Should we use the ciphersuite-specified endianess (i.e. just call `serialize`) so that the caller can deserialize it if they wish?

Reply:
OK I had a shot at this in 530b5ee

- Switches to `Vec<u8>`
- Uses ciphersuite-endianess
- Introduces `Identifier::deserialize`
- Makes `Identifier::serialize` public (my rationale is that people will need to serialize/deserialize identifiers for communicating)

Keeping it little-endian would require adding `Identifier::(de)serialize_little_endian` which is more awkward.

What do you think?